lib/cartopy/mpl/geoaxes.py

```diff
@@ -417,6 +417,51 @@ def coastlines(self, resolution='110m', color='black', **kwargs):
                                                       resolution, **kwargs)
         return self.add_feature(feature)
 
+    def tissot(self, rad_km=5e5, n_samples=80, lon_n=6, lat_n=6, lon_max=180,
```

Comment:
PlateCarree will be a spherical approximation, from memory.

> Did you mean to change the radius to 5000?

Nope. Thanks for asking!